src/compiler/scala/tools/nsc/GenericRunnerSettings.scala

```diff
@@ -16,8 +16,9 @@ class GenericRunnerSettings(error: String => Unit) extends Settings(error) {
       "-howtorun",
       "how",
       "how to run the specified code",
-      List("object", "script", "jar", "repl", "guess"),
+      List("object", "script", "jar", "repl", "fsc", "guess"),
```

Comment:
I think `-howtorun:fsc` is not easy to understand. Maybe `script-compdaemon` is more clear?

Reply:
I considered `-howtorun:resident` because `fsc` is a brand name. When `fsc` is a module, it will need the brand exposure, but `resident` is generic; ideally, there will be multiple REPL frontends and backends, and some of the backends will support `resident` flag. Or am I getting ahead of myself?

Reply:
I think `-nc:false` just works.

Reply:
Ah... Here's what tripped me up:

```
➜  sandbox git:(inlineRounds) scala -nc Test.scala
hi
➜  sandbox git:(inlineRounds) fsc -shutdown
[No compilation server running.]
➜  sandbox git:(inlineRounds) scala Test.scala -nc
hi
➜  sandbox git:(inlineRounds) fsc -shutdown
[Compile server exited]
➜  sandbox git:(inlineRounds) fsc -shutdown
[No compilation server running.]
➜  sandbox git:(inlineRounds)
```

Reply:
I'm doing a quick refactor of ScriptRunner to break out the functionality, and then `-Yscriptrunner:com.github.somsnytt.BestScriptRunnerEver`.

Reply:
I remember there's a comment in settings.process about handling trailing options; I'll look if it's broken for Generic...Settings.